### PR TITLE
e2e: reduce redundant syncer test logs

### PR DESCRIPTION
**Pull request by sttts**
Only print new pod events.

### CODE REVIEW

**Comment by sttts** [comment]
@jmprusi

**Comment by sttts** [comment]
flake, probably https://github.com/kcp-dev/kcp/pull/1355

File: test/e2e/framework/fixture.go [hunk 1]
```diff
@@ -609,20 +609,17 @@ func (sf *StartedSyncerFixture) WaitForClusterReadyReason(t *testing.T, ctx cont
 	kcpClusterClient, err := kcpclient.NewClusterForConfig(cfg.UpstreamConfig)
 	require.NoError(t, err)
 	kcpClient := kcpClusterClient.Cluster(cfg.KCPClusterName)
-	require.Eventually(t, func() bool {
+	Eventually(t, func() (bool, string) {
```
Comment (s-urbaniak) [inline test/e2e/framework/fixture.go:612]: not related to here: can we document `Eventually` and the pitfalls it solves vs. using `require.Eventually` directly?

Comment (sttts) [reply test/e2e/framework/fixture.go:612]: done as part of other PR.

**Pull request merged**